plugins/vpp/l3plugin/vppdump/dump_vppcalls.go

```diff
@@ -101,33 +101,35 @@ func dumpStaticRouteIPDetails(tableID uint32, tableName []byte, address []byte,
 		ipAddr = fmt.Sprintf("%s/%d", net.IP(address[:4]).To4().String(), uint32(prefixLen))
 	}
 
-	rt := &vppcalls.Route{}
+	rts := []*vppcalls.Route{}
```

Comment:
No need to initialize the slice. You can just keep it nil: `var rts []*vppcalls.Router`.

The `append` function will take care of initializing it.